src/modules/ce/components/unit/SourceEnrollmentCard.tsx

```diff
@@ -0,0 +1,159 @@
+import { Alert, Divider, Stack, Typography } from '@mui/material';
+import React, { useId } from 'react';
+import ButtonLink from '@/components/elements/ButtonLink';
+import CommonSelectableCard from '@/components/elements/CommonSelectableCard';
+import CommonTextWithIcon from '@/components/elements/CommonTextWithIcon';
+import CommonTruncatedList from '@/components/elements/CommonTruncatedList';
+import RelativeDateDisplay from '@/components/elements/RelativeDateDisplay';
+import RouterLink from '@/components/elements/RouterLink';
+import {
+  ClientIcon,
+  DateRangeIcon,
+  ProjectIcon,
+} from '@/components/elements/SemanticIcons';
+import EnrollmentDateRangeWithStatus from '@/modules/hmis/components/EnrollmentDateRangeWithStatus';
+import HmisEnum from '@/modules/hmis/components/HmisEnum';
+import { EnrollmentDashboardRoutes } from '@/routes/routes';
+import { HmisEnums } from '@/types/gqlEnums';
+import {
+  CeReferralSourceEnrollmentFieldsFragment,
+  RelationshipToHoH,
+} from '@/types/gqlTypes';
+import { generateSafePath } from '@/utils/pathEncoding';
+
+interface Props {
+  enrollment: CeReferralSourceEnrollmentFieldsFragment;
+  selected: boolean;
+  onSelect: () => void;
+}
+
+const SourceEnrollmentCard: React.FC<Props> = ({
+  enrollment,
+  selected,
+  onSelect,
+}) => {
+  const radioId = useId();
+
+  // will always be true for now (otherwise the enrollment wouldn't have been resolved),
+  // but in the future we will resolve enrollments here that user may not be able to view,
+  // including enrollments in different data sources
+  const includeLinks =
+    enrollment.dataSource.isCurrentDataSource &&
+    enrollment.access.canViewEnrollmentDetails;
+
+  return (
+    <CommonSelectableCard
+      key={enrollment.id}
+      value={enrollment.id}
+      selected={selected}
+      onSelect={onSelect}
+      ariaLabelledBy={radioId}
```

Comment:
Just noting this isn't perfect from an a11y perspective, because the whole text of the card will be read by screenreader when the user is navigating between radio options. I think it's acceptable for mvp, since the user may need to know all that information to make a decision.